src/index.ts

```diff
@@ -293,16 +298,16 @@ export class Chart extends EventEmitter.EventEmitter {
     this.buildAxis()
     this.buildAxisLabel()
 
-    // draw each datum after the wrapper was set up
```

Comment:
I'm not sure if this would work just fine, if you got the chance to build the project did you check the examples in the demo page?

Reply:
I have tested it in my own project that uses function-plot and it works fine, but I can do this too

Reply:
It worked fine, but I moved the zoom helper element inside the svg to the top just for good measure.